src/PickupPoint/PickupPoint.php

```diff
@@ -17,270 +17,270 @@ class PickupPoint {
 	use JsonFormat;
 	use ArrayFormat;
 
-    #region Properties
-    /**
-     * ID of the pickup point or reference needed for the shipping plugin.
-     *
-     * @var string
-     */
+	// region Properties
```

Comment:
This should still be a # not a comment. It makes it collapsible in most IDEs for the entire list of properties. The same needs to be reverted for all other regions and end regions that where changed